packages/core/src/routes/interaction/utils/user-display-name.ts

```diff
@@ -0,0 +1,13 @@
+import { type User } from '@logto/schemas';
+
+/**
+ * Get user display name from multiple fields
+ */
+export const getUserDisplayName = ({
```

Comment:
nit: maybe we should put it in `@logto/shared` package

Reply:
updated